src/systemd-sonic-generator/systemd-sonic-generator.c

```diff
@@ -12,10 +12,12 @@
 #define MAX_NUM_TARGETS 5
 #define MAX_NUM_INSTALL_LINES 5
 #define MAX_NUM_UNITS 128
+#define NUM_ASICS 6
```

Comment:
why hard code here?

Reply:
I'm not really sure how else to get this information to the generator - later on it can probably read from a file somewhere.